src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -28,7 +25,7 @@ public DefaultTagHelperResolver(ErrorReporter errorReporter, Workspace workspace
             _workspace = workspace;
         }
 
-        public async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project project)
+        public override async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project project, CancellationToken cancellationToken)
```

Comment:
Shouldn't this CancellationToken be passed in to all the async calls below?

Reply:
I see the same issue here https://github.com/aspnet/Razor/blob/dev/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotWorker.cs#L51

Reply:
Going to skip out on the second part because that would require us to pass the cancellation token as part of `ProjectSnapshotUpdateContext`.